tests/subsys/dfu/fmfu_fdev/merged.hex

```diff
@@ -0,0 +1,4890 @@
+:020000040000FA
```

Comment:
Why are we committing hex files?

Reply:
As stated in the description of the pull request, skip the tests as there are not done ;)

To answer your question, we might use hex files to flash and verify signatures etc.

Reply:
Oops, sorry about that. I tried to be good about avoiding review on `tests/**/*` files, but apparently overlooked this one. Probably because it was a hex.

Reply:
Hehe, you have to patch your inner regex engine